ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Run.hs

```diff
@@ -272,7 +274,9 @@ runPointSchedule schedulerConfig genesisTest tracer0 =
           = pure nullTracer
     stateTracer <- mkStateTracer
     BlockFetch.startBlockFetchLogic registry chainDb fetchClientRegistry getCandidates
-    void $ forkLinkedThread registry "ChainSel trigger" (reprocessLoEBlocksOnCandidateChange chainDb getCandidates)
+    -- void $ forkLinkedThread registry "ChainSel trigger" (reprocessLoEBlocksOnCandidateChange chainDb getCandidates)
```

Comment:
Remove the comment. (?)
Is `reprocessLoEBlocksOnCandidateChange` dead code now?

Reply:
I deleted the function